registry.k8s.io/images/k8s-staging-cloud-provider-gcp/generate.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+readonly repo="gcr.io/k8s-staging-cloud-provider-gcp"
+readonly tag_filter="tags~^v\d+\.\d+\.\d+\$"
+# List of repos under https://console.cloud.google.com/gcr/images/k8s-staging-cloud-provider-gcp/GLOBAL
+readonly images=(
+    gcp-compute-persistent-disk-csi-driver
+    gcp-filestore-csi-driver
+    cloud-controller-manager
+)
+
+for image in "${images[@]}"; do
+    echo "- name: ${image}"
+    echo "  dmap:"
+    gcloud container images list-tags \
+        "${repo}/$image" \
+        --format="get(digest, tags)" \
+        --sort-by="tags" \
```

Comment:
too bad we can't sort by version number in the tags, the lexicographic ordering is annoying.

Reply:
With some bash foo it could probably be resorted but I'll leave that as  future improvement :-)